nodejs/packages/layer/package.json

```diff
@@ -27,24 +27,24 @@
   },
   "dependencies": {
     "@opentelemetry/api": "^1.7.0",
-    "@opentelemetry/exporter-trace-otlp-proto": "^0.50.0",
-    "@opentelemetry/exporter-metrics-otlp-proto": "^0.50.0",
-    "@opentelemetry/instrumentation": "^0.50.0",
-    "@opentelemetry/instrumentation-aws-lambda": "^0.40.0",
-    "@opentelemetry/instrumentation-aws-sdk": "^0.40.0",
-    "@opentelemetry/instrumentation-dns": "^0.35.0",
-    "@opentelemetry/instrumentation-express": "^0.37.0",
-    "@opentelemetry/instrumentation-graphql": "^0.39.0",
-    "@opentelemetry/instrumentation-grpc": "^0.50.0",
-    "@opentelemetry/instrumentation-hapi": "^0.36.0",
-    "@opentelemetry/instrumentation-http": "^0.50.0",
-    "@opentelemetry/instrumentation-ioredis": "^0.39.0",
-    "@opentelemetry/instrumentation-koa": "^0.39.0",
-    "@opentelemetry/instrumentation-mongodb": "^0.42.0",
-    "@opentelemetry/instrumentation-mysql": "^0.37.0",
-    "@opentelemetry/instrumentation-net": "^0.35.0",
-    "@opentelemetry/instrumentation-pg": "^0.40.0",
-    "@opentelemetry/instrumentation-redis": "^0.38.0",
+    "@opentelemetry/exporter-trace-otlp-proto": "^0.51.0",
+    "@opentelemetry/exporter-metrics-otlp-proto": "^0.51.0",
+    "@opentelemetry/instrumentation": "^0.51.0",
+    "@opentelemetry/instrumentation-aws-lambda": "^0.41.0",
+    "@opentelemetry/instrumentation-aws-sdk": "^0.41.0",
+    "@opentelemetry/instrumentation-dns": "^0.36.0",
+    "@opentelemetry/instrumentation-express": "^0.38.0",
+    "@opentelemetry/instrumentation-graphql": "^0.40.0",
+    "@opentelemetry/instrumentation-grpc": "^0.51.0",
+    "@opentelemetry/instrumentation-hapi": "^0.37.0",
+    "@opentelemetry/instrumentation-http": "^0.51.0",
+    "@opentelemetry/instrumentation-ioredis": "^0.40.0",
+    "@opentelemetry/instrumentation-koa": "^0.40.0",
+    "@opentelemetry/instrumentation-mongodb": "^0.43.0",
+    "@opentelemetry/instrumentation-mysql": "^0.38.0",
+    "@opentelemetry/instrumentation-net": "^0.36.0",
+    "@opentelemetry/instrumentation-pg": "^0.41.0",
+    "@opentelemetry/instrumentation-redis": "^0.39.0",
```

Comment:
I'm confused as to why these were grouped under `other` instead of  a separate `opentelemetry` PR.  @vasireddy99 @rapphil @nslaughter any ideas?

Reply:
I guess it got fixed somehow: #1298